bin/indexer

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const execa = require('execa');
+const path = require('path');
+const os = require('os');
+
+const REPO_PATH = 'tmp';
+
+(async () => {
+    let indexPath = path.join(REPO_PATH, "index.js")
+    while (true) {
+        console.log('at=main level=info msg="trying to update registry-index repo"');
+        const success = await createUpdateRepo(REPO_PATH);
+        if (success) {
+            console.log('at=main level=info msg="building index config data cache"');
+            fs.writeFileSync(indexPath, JSON.stringify(getNormalisedRegistry(REPO_PATH)))
+            console.log('at=main level=info msg="serialized index to file"');
+            await buildIndex(indexPath)
+            console.log('at=main level=debug msg="done building index config data cache"');
+        } else {
+            console.log('at=main level=warn msg="failed to update registry-index repo"');
+        }
+        await sleep(300000); // wait 5 minutes
+    }
+})();
+
+function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+async function buildIndex(indexPath) {
+    try {
+        let subProc = execa('./bin/index-buildpacks', [indexPath])
```

Comment:
Hey Joe, I was looking at this late Friday, and I'm pretty sure we need a `await` in front of this.

Reply:
I'm not sure, but I think the `await` is a few lines below. This way, it sets up the std/stderr pipes first.

Reply:
ah, I missed that 👍🏼

Reply:
Yup, that's fine.   I must have been looking at an older commit.  This should be fine.